modules/dcache/src/main/java/org/dcache/pool/classic/IoQueueManager.java

```diff
@@ -406,6 +413,9 @@ public synchronized String call() throws IllegalArgumentException, NoSuchElement
         @Option(name = "r", usage = "Sort output in reverse order.")
         boolean reverseSort;
 
+        @Option(name = "c", usage = "Filter output by client IP or hostname.")
```

Comment:
`c` is OK, but I think we can be more generous with our name!  perhaps `client` isn't unreasonable, here.  Incidentally, the others (`r`, `S` and `t`) are single letters because this is how the Unix `ls` command works.